### PR TITLE
Add console and logs options to make for aks

### PR DESCRIPTION
**Pull request by RMcVelia**
### Context

Add Makefile options to connect to the console or display logs for an aks deployed app 

### Changes proposed in this pull request

Makefile changes
- add aks-console, aks-logs, aks-ssh and aks-worker-ssh options

### Guidance to review

make review_aks aks-logs APP_NAME=pr-nnnn
make review_aks aks-console APP_NAME=pr-nnnn
make review_aks aks-ssh APP_NAME=pr-nnnn
make review_aks aks-worker-ssh APP_NAME=pr-nnnn

### Important business

- [ ] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [ ] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?
- [ ] Do we need to send any updates to DQT as part of the work in this PR?
- [ ] Does this PR need an ADR?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -215,6 +220,17 @@ terraform-init:
 	az account set -s $(AZ_SUBSCRIPTION) && az account show \
 	&& terraform -chdir=terraform/$(PLATFORM) init -reconfigure -backend-config=./workspace-variables/$(DEPLOY_ENV)_backend.tfvars $(backend_key)
 
+get-cluster-credentials: set-azure-account ## make <config> get-cluster-credentials [ENVIRONMENT=<clusterX>]
+	az aks get-credentials --overwrite-existing -g ${RESOURCE_NAME_PREFIX}-tsc-${CLUSTER_SHORT}-rg -n ${RESOURCE_NAME_PREFIX}-tsc-${CLUSTER}-aks
+
+aks-console: read-cluster-config get-cluster-credentials
+	$(if $(APP_NAME), $(eval export APP_ID=$(APP_NAME)) , $(eval export APP_ID=$(CONFIG_LONG)))
+	kubectl -n ${NAMESPACE} exec -ti --tty deployment/register-${APP_ID} -- /bin/sh
```
Comment (saliceti) [inline Makefile:228]: Can we run the rails console instead? (Similar to the `console:` target)

File: Makefile [hunk 2]
```diff
@@ -215,6 +220,17 @@ terraform-init:
...
+
+aks-logs: read-cluster-config get-cluster-credentials
+	$(if $(APP_NAME), $(eval export APP_ID=$(APP_NAME)) , $(eval export APP_ID=$(CONFIG_LONG)))
+	kubectl -n ${NAMESPACE} logs deployment/register-${APP_ID} --timestamps=true
```
Comment (saliceti) [inline Makefile:232]: This shows the logs from the first pod in the deployment. To get all the pods we could use something like: `kubectl -n ${NAMESPACE} logs -l app=register-${APP_ID} --timestamps=true`

File: Makefile (hunk 1 above)
Comment (RMcVelia) [reply Makefile]: changed to run rails console

File: Makefile (hunk 2 above)
Comment (RMcVelia) [reply Makefile]: changed to use -l

**Review by saliceti — APPROVED** [review]

**Pull request merged**